glide-core/redis-rs/redis/src/types.rs

```diff
@@ -585,6 +518,41 @@ impl Value {
             _ => Err(self),
         }
     }
+
+    /// If value contains a server error, return it as an Err. Otherwise wrap the value in Ok.
```

Comment:
please add decription - if there're multiple errors it will only return the first one.
Write when should this function be used